lib/util.js

```diff
@@ -37,6 +38,9 @@ util.cleanAnonymous = async () => {
         const arangoDbExists = await arango.checkIfDatabaseExists(username);
         if (arangoDbExists) await arango.deleteAccount(username);
 
+        const influxDbExists = await influx.checkAccount(username);
+        if (influxDbExists) await influx.deleteAccount(user);
```

Comment:
You have the choice to pass in `username`, so you should do that instead of `user`.  I don't think it makes sense to pass in an object to a function that only uses one `string data-type` variable as an input. That is confusing -- when I look at the function parameters, it makes me think the function needs something else from `user` other than just `user.username`

Reply:
Basically, this code threw me off. PG and Elastic use username for checks and user for deletions while Arango needs only username in both cases. I was trying to be consistent so I added user as argument for Influx. 
```
    const pgDbExists = await pg.userHasPgAccount(username);
    if (pgDbExists) await pg.deletePgAccount(user);

    const esDbExists = await es.checkAccount(username);
    if (esDbExists) await es.deleteAccount(user);

    const arangoDbExists = await arango.checkIfDatabaseExists(username);
    if (arangoDbExists) await arango.deleteAccount(username);
```
I changed it, so deleteAccount now needs only username as argument.